### PR TITLE
Kotlin CleanUp of ImportTest.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin CleanUp of ImportTest.kt

## How Has This Been Tested?
Tested on a physical device

### CODE REVIEW

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt [hunk 1]
```diff
@@ -41,12 +41,10 @@ import java.io.File
 import java.io.FileOutputStream
 import java.io.IOException
 
-@KotlinCleanup("is -> equalTo")
 @KotlinCleanup("IDE Lint")
 @RunWith(AndroidJUnit4::class)
 class ImportTest : InstrumentedTest() {
-    @KotlinCleanup("init here/lateinit")
-    private var mTestCol: Collection? = null
+    private lateinit var mTestCol: Collection
```
Comment (ShridharGoel) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt:47]: Also change variable name to `testCol`.

**Review by ShridharGoel — DISMISSED** [review]

**Review by lukstbit — APPROVED** [review]
Consider this an approval once @ShridharGoel request is addressed.

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt (hunk 1 above)
Comment (n1snt) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt]: Any reason why m needs to be removed?

Comment (Prince-kushwaha) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt]: because of good variable naming convention

**Review by BrayanDSO — APPROVED** [review]

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt (hunk 1 above)
Comment (n1snt) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt]: Ok cool. I did not know that. :|

Comment (ShridharGoel) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/tests/libanki/ImportTest.kt]: @n1snt We're trying to move away from Hungarian notation.

**Pull request merged**